experimental/addons/postgres-operator/definitions/traits/postgres-expose.cue

```diff
@@ -0,0 +1,37 @@
+"postgres-expose": {
```

Comment:
can we reuse the expose trait?

Reply:
@wonderflow I've updated ` postgres-expose` trait as `expose` trait. Please reivew it again.

Reply:
why do we need to create a new one instead of using `expose` trait?